homeassistant/components/modbus/cover.py

```diff
@@ -47,6 +50,11 @@ async def async_setup_platform(
 ):
     """Read configuration and create Modbus cover."""
     if discovery_info is None:
+        _LOGGER.error(
```

Comment:
We usually do warning level for these.
```suggestion
        _LOGGER.warning(
```

Reply:
The real question is "can this happen", load_platform is called from modbus.py and there config is passed to be discovery_info so with the new configuration style this cannot be None, but of course it does no real harm.

Reply:
It can happen if they still have the old style config with platforms. I guess that's when we want to warn the user that they need to upgrade their config.